torch/csrc/jit/passes/python_print.cpp

```diff
@@ -1039,6 +1048,33 @@ struct PythonPrintPass {
           stmt << field_stream.str() << ")";
         }
       } break;
+      case prim::CallFunction: {
+        const auto& fn = node->inputs().at(0)->type()->expect<FunctionType>();
+        registerDependency(fn);
+
+        stmt << fn->name()->qualifiedName() << "(";
+        for (size_t i = 1; i < node->inputs().size(); i++) {
+          stmt << useOf(node->inputs()[i]) << ", ";
+        }
+        stmt << ")";
+      } break;
+      case prim::CallMethod: {
+        const auto& self = node->inputs().at(0);
+        const auto& selfType = self->type()->expect<ClassType>();
+        const auto& methodName = node->s(attr::name);
+        const auto method = selfType->getMethod(node->s(attr::name));
+        registerDependency(selfType);
+
+        TORCH_INTERNAL_ASSERT(
+            method->qualname() ==
+            QualifiedName(selfType->name()->qualifiedName(), methodName));
+
+        stmt << useOf(self) << "." << methodName << "(";
```

Comment:
Is there any operator we print inline that is lower precedence than `.`? If so we need to put parentheses around self.

Reply:
I guess might as well just to be safe